factory/defaults/chains/mainnet.yaml

```diff
@@ -513,3 +513,16 @@ chains:
     decimals: 6
     indexer_type: ripple
     polling_period: 3m
+  BASE:
```

Comment:
We should add DTI identifiers for chains on mainnet.  Also need to update XRP.

Please add a comment at the top of this file saying we can get DTI from https://download.dtif.org/data.json.
This way we can remember where to look :)

XRP: L6GTZC9G4
BASE: 6M4WCXL3X
NOBLE: (not released yet, leave blank)